emails/file-deletion.tsx

```diff
@@ -0,0 +1,59 @@
+import {
```

Comment:
wasn't sure if we should keep the text block introducing the time threshold for expiry

Reply:
Could you clarify this?

Reply:
In an ongoing effort to keep WATcloud's file storage efficient and maintain optimal performance, we routinely clean up specific directories that haven't been accessed in the past {daysForExpiry} days. We previously sent an email that you had files which met this criteria. Since they were not accessed by the given deadline, those files have been deleted.

was just trying to check if this was good!

Reply:
Ah I see. LGTM!